parser/prism/Translator.cc

```diff
@@ -114,6 +114,24 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
         return nullptr;
 
     switch (PM_NODE_TYPE(node)) {
+        case PM_ALIAS_GLOBAL_VARIABLE_NODE: { // // The `alias` keyword used for global vars, like `alias $new $old`
```

Comment:
Never knew we can alias global variables 🤯

Reply:
But not instance or class variables... weird.